.github/workflows/validation.yml

```diff
@@ -199,6 +199,7 @@ jobs:
               -Dmaven.wagon.http.retryHandler.count=3 \
               -P\!it-modules \
               -Pit-modules-${{ matrix.it-modules }} \
+              -Pproduction \
```

Comment:
Is the PR title wrong or this profile activation?

Reply:
Well the intent is to run them in dev mode when you run locally, so you can develop, and in production mode in CI so it is faster